lca-cli/postpivot/postpivot.go

```diff
@@ -377,8 +380,8 @@ func (p *PostPivot) setNewClusterID(ctx context.Context, client runtimeclient.Cl
 	}
 
 	if seedReconfiguration.ClusterID == "" {
```

Comment:
This would be a fatal error on IBU.

Reply:
In IBU you will always have the clusterID as part of the config, otherwise it should fail during prep.
TBH, this change is not a must for IBI, we plan to generate the cluster ID in IBIO (we just didn't figure out where to store it yet).